app/conf/Configuration.scala

```diff
@@ -229,6 +229,7 @@ class ApplicationConfiguration(val playConfiguration: PlayConfiguration, val isP
   object faciatool {
     lazy val breakingNewsFront = "breaking-news"
     lazy val frontPressToolQueue = getString("frontpress.sqs.tool_queue_url")
+    lazy val issuePublishedEventsQueue = getString("issue_published_events.queue_url").getOrElse("")
```

Comment:
Should we throw if we're unable to get this value from config?

Reply:
yeah we can use .get that will throw exception

Reply:
done